packages/multichain-api-middleware/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Bump `@metamask/chain-agnostic-permission` from `1.0.0` to `1.1.0` ([#6241](https://github.com/MetaMask/core/pull/6241))
+- Bump `@metamask/chain-agnostic-permission` from `1.1.0` to `1.1.1` ([#6241](https://github.com/MetaMask/core/pull/6241), [#6345](https://github.com/MetaMask/core/pull/6241))
```

Comment:
Nit: Suggest rolling back the from version to the previous one since this package hasn’t been released yet.
```suggestion
- Bump `@metamask/chain-agnostic-permission` from `1.0.0` to `1.1.1` ([#6241](https://github.com/MetaMask/core/pull/6241), [#6345](https://github.com/MetaMask/core/pull/6241))